collectors/plugins.d/plugins_d.c

```diff
@@ -544,47 +608,12 @@ void *pluginsd_worker_thread(void *arg) {
         // get the return code
         int code = mypclose(fp, cd->pid);
```

Comment:
i suggest to remove comment and rename `code` to `ret_code`. Adding context via variable name >> using comment

Reply:
A whatever name it should be. Gist is

> Adding context via variable name >> using comment

Reply:
Right, done!